server/core/src/notify_service_manager.cpp

```diff
@@ -0,0 +1,102 @@
+#include "irods/notify_service_manager.hpp"
+
+/// \file
```

Comment:
This can be removed since it's a cpp file.

Reply:
Fixed